### ISSUE TITLE
Naming convention

### ISSUE THREAD

**Issue by icholy** [bradylove/envstruct#5]
This is just a pet peve, but `Ip` should be `IP`. 

``` go
type HostInfo struct {
    Ip       string `env:"host_ip,required"`
    Password string `env:"password,noreport"`
    Port     int    `env:"host_port"`
}
```

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

**Comment by bradylove** [comment]
Thanks for the link, I will fix this with the other issue you submitted.

**Comment by bradylove** [comment]
This has been fixed in 94ecb16621c2dc6bf22c029a3ff2e286e95961bd
